recipes/qt/5.x.x/conanfile.py

```diff
@@ -111,6 +112,7 @@ class QtConan(ConanFile):
         "with_openal": True,
         "with_zstd": True,
         "with_gstreamer": False,
+        "with_dbus": False,
```

Comment:
My understanding is that it was always active before. Why changing the default?

Reply:
To put it shortly, because we don't have a working dbus package on mac and Windows, and the existing qt package relies on dynamic loading of dbus library, which is an antipattern for a package manager IMHO.